Junk4Dollars-app/Auctions/AuctionListView.swift

```diff
@@ -0,0 +1,4 @@
+import UIKit
+public protocol AuctionListView: UIViewController {
```

Comment:
Why does this inherit from `UIViewController`? I would try removing that and seeing if it still works.

Reply:
It looked like having a protocol inherit from something enforces that anything that conforms to that protocol must also inherit from that same class. It'll definitely work without it though.